ocaml-general/Dockerfile

```
@@ -25,29 +25,11 @@ RUN apt-get update && apt-get install -y \
   opam
 
 ARG OCAML_VERSION=4.13.1
```

```suggestion
ARG OCAML_VERSION=5.0.0
```
OCaml 5 対応したら default version は 5 にたい 🙏

---

https://github.com/kxcteam/pubinfra.dockerfiles/pull/27/commits/4e337a062c2c9b867298e042ed9050e8644c7a07